humans.txt

```diff
@@ -0,0 +1,32 @@
+/* TEAM */
+
+	Chef: Jaydson Gomes
```

Comment:
Chef = cooker.
"Leader", "Founder" or "Maintainer" would be more appropriate I guess.
cc @jaydson